geoengine/datasets.py

```diff
@@ -444,6 +412,7 @@ def upload_dataframe(
     ints = [key for (key, value) in columns.items() if value['dataType'] == 'int']
     texts = [key for (key, value) in columns.items() if value['dataType'] == 'text']
 
+    # TODO: use API dict to model the request
```

Comment:
Since the original task was to work on datasets, shouldn't you do this here then?

Reply:
[d3dcede](https://github.com/geo-engine/geoengine-python/pull/94/commits/d3dcede662c92a6096fb59a64ba6c36f0327bc84)